### PR TITLE
@kanaabe: Set time in published_at

### PR DESCRIPTION
**Pull request by jtotoole**

### CODE REVIEW

File: client/apps/edit/components/admin/index.coffee [hunk 1]
```diff
@@ -349,13 +349,15 @@ module.exports = class EditAdmin extends Backbone.View
...
-      @formatAndSetPublishDate($('.edit-admin-input-date').val())
+    $('.edit-admin-input-date, .edit-admin-input-time').on 'blur', =>
+      dateAndTime = @$('.edit-admin-input-date').val() + ' ' + @$('.edit-admin-input-time').val()
+      @formatAndSetPublishDate dateAndTime
       @article.save published_at: @saveFormat
       false
     @formatAndSetPublishDate @article.get('published_at')
 
   formatAndSetPublishDate: (date) ->
+    date = new Date(date).toISOString()
```
Comment (jtotoole) [inline client/apps/edit/components/admin/index.coffee]: Moment was throwing a deprecation warning when I ran the test initially bc I guess they're not going to be as liberal in the formatting they'll accept to convert in the next release 
screen shot 2015-12-04 at 6 06 59 pm

Adding this line resolved it

Comment (kanaabe) [inline client/apps/edit/components/admin/index.coffee]: :angel:

**Comment by kanaabe** [comment]
This looks :gem:

**Comment by jtotoole** [comment]
@kanaabe fyi we're bumping 'lose yourself' at happy hour rn

**Comment by kanaabe** [comment]
I am incredibly jealous :8ball:

**Pull request merged**